packages/opencensus-propagation-b3/src/validators.ts

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
This file is taken from ```opencensus-propagation-tracecontext``` -> https://github.com/census-instrumentation/opencensus-node/blob/master/packages/opencensus-propagation-tracecontext/src/validators.ts